tests/testthat/test_RT_functions.R

```diff
@@ -10,24 +11,11 @@ test_that('we can create an RT ticket', {
   }
   
   db <- create_dummy_database()
-  ticket <- create_ticket(db$id, db$pi_email)
   
-  expect_type(ticket, 'character')
-})
-
-test_that('create_ticket errors gracefully', {
-  if (check_rt_login(rt_url)) {
-    skip('Logged in to RT. Test will probably not pass')
-  }
-  if (Sys.getenv('SLACK_WEBHOOK_URL') == '') {
-    skip('Run slackr_setup() to run this test')
-  }
-  
-  db <- create_dummy_database()
+  #does not return ticket but creates ticket
```

Comment:
If `create_ticket()` doesn't return the ticket, what does it return? Instead of removing the functionality, can we get the ticket based on what is returned (like the `id`)?

Reply:
I removed the part of this test that involves the same rt problem with `rt_ticket_history_reply` in the `create_ticket` code where it no longer gives the error code. Without the error code, I can't check if it creates the `rt_ticket_create_error` object. So now I have changed the test to see if a ticket to see if `create_ticket` indeed returns some character object (the ticket `id`)

Reply:
(Jump jumping in here with a note on the change in the `rt` package behavior we introduced for the first release: If `rt_ticket_history_reply` fails, it _should_ throw a `stop` which can be `tryCatch`'d. The error message, if one was sent back by RT, will be available in the `tryCatch`. If it's not, let us know over on https://github.com/NCEAS/rt and we can put in a workaround).

Reply:
Oh nice I will look into that! I should have asked earlier!

Reply:
Thanks @amoeba !